Added Warning for WIFI_SSID with special chars or empty spaces (#466)
project-chip/certification-tool: docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc

* *$PATH_TO_PAA_ROOTS*: Path on host where PAA roots are located. Failure to provide a correct path will cause early failure during commissioning (e.g., /var/paa-root-certs/)
* *$DISCRIMINATOR*: Long discriminator for DUT (e.g., 3840 for Linux examples)
* *$SETUP_PASSCODE*: Setup passcode for DUT (e.g., 20202021 for Linux examples)
* *$WIFI_SSID*: SSID of Wi-Fi AP to which to attempt connection 
+

> WARNING: Currently, WIFI_SSID with special chars or empty spaces are not supported.

Reviewer: ```suggestion
WARNING: Currently, WIFI_SSID with special characters or empty spaces is not supported.
```
